consistent_web_main/main.go

```diff
@@ -185,7 +186,7 @@ func recordLatency(latency time.Duration) {
 }
 
 func init() {
-	latencyFileName = time.Now().String() + ".txt"
+	latencyFileName = "./test.txt"
```

Comment:
should we revert this but maybe use unix time to make sure there isn't any weird characters in the name?